.github/workflows/benchmark_gpu_core_crypto.yml

```diff
@@ -56,14 +56,15 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt update
-          sudo apt install -y checkinstall zlib1g-dev libssl-dev
+          sudo apt install -y checkinstall zlib1g-dev libssl-dev libclang-dev
```

Comment:
could be made part of the AMI potentially to avoid apt hangs in the CI, cc @soonum wdyt ?

hopefull doable with hyperstack ?

Reply:
We can't have AMIs with Hyperstack unfortunately, maybe in the future, so we always install dependencies with apt.

Reply:
got it